shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -79,7 +79,7 @@
   // Since each virtual display has it's unique context this allows associating any view with the
   // platform view that
   // it is associated with(e.g if a platform view creates other views in the same virtual display.
-  private final HashMap<Context, View> contextToPlatformView;
+  @VisibleForTesting /* package */ final HashMap<Context, View> contextToPlatformView;
```

Comment:
nit: Is /* package */ necessary? This should be the implied visibility right?

Reply:
it increases readability since the Java language isn't very descriptive.  Debatable, but it's commonly used in g3 as well as in the engine.